Formula/geph2.rb

```diff
@@ -0,0 +1,25 @@
+class Geph2 < Formula
+  desc "Modular Internet censorship circumvention system"
+  homepage "https://geph.io"
+  url "https://github.com/geph-official/geph2/archive/v0.20.2.tar.gz"
+  sha256 "fe5bef5ed04ab6f2e254e7e142ce93d57f5f1c157651db2c29aaf3cf13abf647"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    bin_path = buildpath/"src/github.com/geph-official/geph2"
+    bin_path.install Dir["*"]
+    cd bin_path/"cmd/geph-client" do
+      ENV["GOOS"] = "darwin"
+      ENV["GOARCH"] = "amd64"
```

Comment:
Is this required for the build?

Reply:
Not actually, it seems only to be necessary on the developer's CI machine which was running Linux and I just copied them without thinking. I've removed them now & Sorry for the inconvenience.